addons/web/static/src/core/utils/xml.js

```diff
@@ -30,6 +30,11 @@ export class XMLParser {
     parseXML(arch) {
         const parser = new DOMParser();
         const xml = parser.parseFromString(arch, "text/xml");
+
+        if (xml.body && xml.body.firstChild && xml.body.firstChild.tagName === "parsererror") {
+            throw new Error(`An error occured while parsing ${arch}: ${xml.body.firstChild}`);
```

Comment:
maybe check https://newbedev.com/how-do-i-detect-xml-parsing-errors-when-using-javascript-s-domparser-in-a-cross-browser-way